docs/content/docs/install/installation.md

```diff
@@ -95,3 +95,29 @@ subjects:
 `Pipelines as Code` provide a CLI which is designed to work as tkn plug-in. To
 install the plug-in follow the instruction from the [CLI](/docs/guide/cli)
 documentation.
+
+## Controller Secure Connection
+
+Pipelines As Code Controller now support both `HTTP` and `HTTPS` connection by adding a secret with tls configurations.
+
+First, create a secret which includes those certificates
+
+```shell
+  kubectl create secret generic -n pipelines-as-code pipelines-as-code-tls-secret \
+    --from-file=cert=/path/to/crt/file \
+    --from-file=key=/path/to/key/file
+```
+
+Next, restart the `pipelines-as-code-controller` pod in `pipelines-as-code` namespace and controller would use those tls configuration.
```

Comment:
```suggestion
Next, restart the `pipelines-as-code-controller` pod in `pipelines-as-code` namespace and when the controller will be restarted it will use the tls secrets.
```